test/boostTest.cpp

```diff
@@ -143,9 +143,9 @@ test_suite* init_unit_test_suite( int /*argc*/, char* /*argv*/[] )
 	bool disableSemantics = !dev::test::EVMHost::getVM(dev::test::Options::get().evmonePath.string());
 	if (disableSemantics)
 	{
-		cout << "Unable to find libevmone.so. Please provide the path using -- --evmonepath <path>." << endl;
+		cout << "Unable to find " EVMONE_FILENAME ". Please provide the path using -- --evmonepath <path>." << endl;
```

Comment:
Actually: should this and the same in ``isoltest`` actually both (1) be extracted to ``test/Common.cpp`` and (2) output to ``cerr`` instead of ``cout``?

Reply:
That's rather part of https://github.com/ethereum/solidity/issues/6432, though, I guess.

Reply:
I think if you output on stderr, it might fail on appveyor.

Reply:
It probably should and we should download/install evmone on appveyor?